e2e/e2etests/test_solana_withdraw.go

```diff
@@ -0,0 +1,43 @@
+package e2etests
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/gagliardetto/solana-go"
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/zetacore/e2e/runner"
+)
+
+func TestSolanaWithdraw(r *runner.E2ERunner, args []string) {
+	require.Len(r, args, 1)
+
+	// print balance of from address
+	solZRC20 := r.SOLZRC20
```

Comment:
Why creating a temporary variable here? It is just simpler and cleaer to directly use `r.SOLZRC20`, it's is only used twice

Reply:
https://github.com/zeta-chain/node/pull/2560/commits/e94fd92e535981bea13dd89a2ad4e13a90a9c147